src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const topSpider =
+  wall.getBoundingClientRect().height / 2 -
+  spider.getBoundingClientRect().height / 2;
+const leftSpider =
+  wall.getBoundingClientRect().width / 2 -
+  spider.getBoundingClientRect().width / 2;
+
+spider.style.top = `${topSpider}px`;
+spider.style.left = `${leftSpider}px`;
```

Comment:
The spider's position is calculated and set when the script loads. If the window is resized after the script loads, the spider will no longer be in the center of the wall. Consider adding an event listener for the window resize event, and update the spider's position in the event handler. This way, the spider will always stay in the center of the wall, even if the window is resized.